client/src/components/InlinePanel/index.js

```diff
@@ -243,5 +267,16 @@ export class InlinePanel extends ExpandingFormset {
       if (this.opts.onAdd) this.opts.onAdd(formIndex);
       if (this.opts.onInit) this.opts.onInit(formIndex);
     }
+
+    // dispatch event for initialising a form
+    const newChild = $(`#inline_child_${newChildPrefix}`).get(0);
+    if (!newChild) return;
+    this.formsElt.children().last().get(0)?.dispatchEvent(
```

Comment:
This newChild check can be removed, it's better to just always dispatch, even if the inline child elements are not being used.